samples/server/petstore/spring-mvc-j8-async/src/main/java/org/openapitools/model/TypeHolderExample.java

```diff
@@ -109,7 +109,7 @@ public TypeHolderExample integerItem(Integer integerItem) {
    * Get integerItem
    * @return integerItem
   */
-  @ApiModelProperty(example = "-2", required = true, value = "")
+  @ApiModelProperty(required = true, value = "")
```

Comment:
these feel like regressions. Aka if it has an example, it should still have one after your change.